client/src/components/inventory/HardwareItem.tsx

```diff
@@ -71,7 +76,7 @@ class HardwareItemBase extends React.Component<Item & HardwareItem, HardwareItem
 
     public finishedLoading() {
         const {toastManager} = this.props;
-        toastManager.add(`Successfully requested ${this.state.qtyRequested}x ${this.props.name}`,
+        toastManager.add(`Successfully requested ${this.state.qtyRequested}x ${this.props.item_name}`,
```

Comment:
why snake_case?

Reply:
It's snake case in the DB so this simplifies the mapping from DB to backend to frontend